packages/web-components/src/menu/menu.stories.ts

```diff
@@ -0,0 +1,53 @@
+import { html } from '@microsoft/fast-element';
+import type { Args, Meta } from '@storybook/html';
+import { renderComponent } from '../helpers.stories.js';
+import type { Menu as FluentMenu } from './menu.js';
+import './define.js';
+import '../menu-item/define.js';
+import '../divider/define.js';
+
+type MenuStoryArgs = Args & FluentMenu;
+type MenuStoryMeta = Meta<MenuStoryArgs>;
+
+const storyTemplate = html<MenuStoryArgs>`
+  <fluent-menu ?open-on-hover="${x => x.openOnHover}">
+    <fluent-menu-button appearance="primary" slot="trigger">Button!</fluent-menu-button>
```

Comment:
Given we'll have a11y testing here, let's use `Toggle Menu` as the label here (and below).